xikolo-ios/Layouts/de.lproj/TabCourses.strings

```diff
@@ -11,9 +11,6 @@
 /* Class = "UISegmentedControl"; CXl-G3-ErQ.segmentTitles[1] = "Find Courses"; ObjectID = "CXl-G3-ErQ"; */
 "CXl-G3-ErQ.segmentTitles[1]" = "Kurse finden";
 
-/* Class = "UILabel"; text = "CourseContent"; ObjectID = "I0c-UK-HJw"; */
-"I0c-UK-HJw.text" = "Kursinhalte";
```

Comment:
Is this wanted?

Reply:
Yes, this label's text will be set "learnings", "discussion" or "course detail". This is just a placeholder I forgot to remove.